db-migrations/000016_remove_duplicate_ip_assignments.up.sql

```diff
@@ -0,0 +1,16 @@
+-- select records we will remove into the new backup table in case we need to recover them
+select aws_private_ip_assignment.id,
```

Comment:
Question: What is the process to clean backup_* table?

Suggestion: It would be good to backup whole database/table and restore, you may see better performance (depending on data size) as well.

Reply:
We will just delete the backup_ table after a certain amount of time and verifying the data looks correct.

This could be overkill, I do have the records exported up to a 30k+ long .sql file that is full of insert statements. I hesitate to check it in as a migration if we feel its too large.

Reply:
We could also just use a snapshot if we really had to rollback but I doubt it would go that far, I would just run the insert statements and that would re-populate the records as it was

Reply:
in terms of record size, it would be inserting around 31,000 records so I don't believe the performance hit would be to significant.

Reply:
discussed in slack, we will do a follow-up task in jira to remove the backup table

Reply:
Yep, that was my concern as well leading me to lean more towards snapshot.